src/fiber/context/arm.cr

```diff
@@ -29,6 +29,15 @@ class Fiber
     #
     # Eventually reset LR to zero to avoid the ARM unwinder to mistake the
     # context switch as a regular call.
+    #
+    # NOTE: depending on the ARM architecture (v7, v6 or older) LLVM uses
+    # different strategies for atomics. By default it uses the "older"
+    # architecture that relies on the libgcc __sync_* symbols; when an armv6 CPU
+    # or +v6 feature is specified it uses the coprocessor instruction as used
+    # below; for armv7 / +v7 it uses the `dmb ish` instruction.
```

Comment:
```suggestion
    # NOTE: depending on the ARM architecture (v7, v6 or older) LLVM uses
    # different strategies for atomics. By default it uses the "older"
    # architecture that relies on the libgcc __sync_* symbols; when an armv6 CPU
    # or +v6 feature is specified it uses the coprocessor instruction as used
    # below, unless the +db (data barrier) feature is set, in which case it
    # uses the `dmb ish` instruction. The +db feature is always enabled since
    # armv7 / +v7.
```